examples/contexttypesbot.py

```diff
@@ -115,17 +114,16 @@ def track_users(update: Update, context: CustomContext) -> None:
 def main() -> None:
     """Run the bot."""
     context_types = ContextTypes(context=CustomContext, chat_data=ChatData)
-    updater = Updater.builder().token("TOKEN").context_types(context_types).build()
+    application = Application.builder().token("TOKEN").context_types(context_types).build()
 
-    application = updater.application
+    application = application.application
```

Comment:
is this right?

Reply:
mh, no. I search & replaced a lot across the examples, didn't check them 1 by 1 yet …